VirtoCommerce.MarketingModule.Web/Scripts/dynamicContent/blades/items/item-detail.js

```diff
@@ -82,6 +82,14 @@
         blade.saveChanges = function () {
             blade.isLoading = true;
 
+            var contentTypeProperty = _.find(blade.currentEntity.dynamicProperties, function(property) {
```

Comment:
Looks like a gag for specific hole.
I'm not sure it is acceptable. 
There is no such problem in v3. Could it be fit for v2?